libtransmission/announcer.cc

```diff
@@ -6,10 +6,11 @@
  *
  */
 
-#include <limits.h> /* INT_MAX */
-#include <stdio.h>
-#include <stdlib.h> /* qsort() */
-#include <string.h> /* strcmp(), memcpy(), strncmp() */
+#include <climits> /* INT_MAX */
+#include <cstdio>
+#include <cstdlib> /* qsort() */
+#include <cstring> /* strcmp(), memcpy(), strncmp() */
+#include <algorithm>
```

Comment:
here and elsewhere, headers in an `#include` group should be sorted

Reply:
Tis done.